src/DGtal/dec/DiscreteExteriorCalculus.h

```diff
@@ -296,15 +296,26 @@ namespace DGtal
     std::string className() const;
 
     /**
+     * Manually insert signed cell into calculus with unit primal and dual size.
+     * Should call updateIndexes() when structure modification is finished.
+     * Be sure to insert all adjacent lower order primal cells.
+     * @param signed_cell the signed cell to be inserted.
+     * @return true if cell was not already inserted, false if only cell was already inserted (cell properties are always updated).
+     */
+    bool
+    insertSCell(const SCell& signed_cell);
+
+    /**
      * Manually insert signed cell into calculus.
      * Should call updateIndexes() when structure modification is finished.
      * Be sure to insert all adjacent lower order primal cells.
      * @param signed_cell the signed cell to be inserted.
-     * @param size_ratio ratio of dual cell size over primal cell size.
+     * @param primal_size size (length, area, volume, ...) of primal cell.
```

Comment:
you should clearly say that for 0-cell, measure or size must be one (asserted in the code)